src/Microsoft.Diagnostics.EventFlow.Outputs.Http/Http.cs

```diff
@@ -0,0 +1,171 @@
+                healthReporter.ReportWarning(errorMessage, EventFlowContextIdentifiers.Configuration);
+                userName = password = null;
+            }
+
+            if (!string.IsNullOrWhiteSpace(userName) && !string.IsNullOrWhiteSpace(password)) 
+            {
+                string httpAuthValue = Convert.ToBase64String(System.Text.ASCIIEncoding.ASCII.GetBytes(string.Format("{0}:{1}", userName, password)));
+                this.httpClient.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Basic", httpAuthValue);
+            }
+
+            switch (this.configuration.ContentType) 
+            {
+                case "text":
+                    if (string.IsNullOrWhiteSpace(this.configuration.HttpContentType))
+                    {
+                        this.configuration.HttpContentType = "text/plain";
+                    }
+                    break;
+
+                case "json":
+                case "json-lines":
+                    if (string.IsNullOrWhiteSpace(this.configuration.HttpContentType))
+                    {
+                        this.configuration.HttpContentType = "application/json";
+                    }
+                    break;
+
+                case "xml":
+                    if (string.IsNullOrWhiteSpace(this.configuration.HttpContentType))
+                    {
+                        this.configuration.HttpContentType = "text/xml";
+                    }
+                    break;
+
+                default:
+                    errorMessage = $"{nameof(configuration)}: unknown ContentType \"{nameof(this.configuration.ContentType)}\" specified";
+                    healthReporter.ReportWarning(errorMessage, EventFlowContextIdentifiers.Configuration);
+                    break;
+            }
+        }
+
+        public Task SendEventsAsync(IReadOnlyCollection<EventData> events, long transmissionSequenceNumber, CancellationToken cancellationToken)
+        {
+            try
+            {
+                string payload = "";
+
+                switch (configuration.ContentType) 
+                {
+                    case "text":
+                        foreach (EventData evt in events)
+                        {
+                            payload += evt.ToString() + "\n";
+                        }
+                        break;
+
+                    case "json":
+                        payload = JsonConvert.SerializeObject(events);
+                        break;
+
+                    case "json-lines":
+                        foreach (EventData evt in events)
+                        {
+                            payload += JsonConvert.SerializeObject(evt) + "\n";
+                        }
+                        break;
+
+                    case "xml":
+                        var serializer = new ConfigurationContainer().Create();
```

Comment:
(minor) Would it make sense to create the serializer during startup and just cache it in a class variable?